projects/storefrontlib/src/layout/a11y/keyboard-focus/config/default-keyboard-focus.config.ts

```diff
@@ -3,5 +3,6 @@ import { KeyboardFocusConfig } from './keyboard-focus.config';
 export const defaultKeyboardFocusConfig: KeyboardFocusConfig = {
   keyboardFocus: {
     enableResetFocusOnNavigate: false,
+    enableResetViewOnNavigate: false,
```

Comment:
Do we _really_ have to change the default config? It's a breaking change in minor version....

Reply:
This does not activate the feature due to it being false. Its just placing the flag in the config to be explicit like the first one.